lib/Dialect/Calyx/Export/CalyxEmitter.cpp

```diff
@@ -142,6 +142,10 @@ struct ImportTracker {
           static constexpr std::string_view sMemories = "memories/seq";
           return {sMemories};
         })
+        .Case<AddFNOp>([&](auto op) -> FailureOr<StringRef> {
```

Comment:
Could you add emitter tests for these please?

Reply:
done!